vllm/engine/arg_utils.py

```diff
@@ -376,8 +364,6 @@ class EngineArgs:
 
     additional_config: Optional[Dict[str, Any]] = None
 
-    # Deprecated, use reasoning_parser instead
-    enable_reasoning: Optional[bool] = None
```

Comment:
We should keep this line (`enable_reasoning: Optional[bool] = None`) while the arg is only deprecated so that users passing the argument don't encounter errors